### ISSUE TITLE
options.block not working

### ISSUE THREAD

**Issue by zzuhan** [shinnn/inline-source-map-comment#1]
```
 if (options.block) {
    return '/* ' + sourceMapBody + ' */';
  }
```

This will generate code like.

```
/* # sourceMappingURL=
```

but the code below only working. 

```
/*# sourceMappingURL=
```

**Comment by shinnn** [comment]
I confirmed the issue. Thanks for reporting!

**Comment by shinnn** [comment]
Fixed in https://github.com/shinnn/inline-source-map-comment/commit/a5089ead78f50a96670541481d8f96c7d7576f24 and released v1.0.5.
